crates/bevy_core_pipeline/src/tonemapping/tonemapping_shared.wgsl

```diff
@@ -35,3 +221,80 @@ fn screen_space_dither(frag_coord: vec2<f32>) -> vec3<f32> {
     dither = fract(dither.rgb / vec3<f32>(103.0, 71.0, 97.0));
     return (dither - 0.5) / 255.0;
 }
+
+fn tone_mapping(in: vec4<f32>) -> vec4<f32> {
+    var color = max(in.rgb, vec3(0.0));
+
+    //let luma = powsafe(vec3(tonemapping_luminance(color)), 1.0); // highlight gain gamma: 0..
+    //color += color * luma.xxx * 1.0; // highlight gain: 0.. 
```

Comment:
```suggestion
```

Reply:
I'd like to maybe keep these for possibly using now or in future